src/map/status.hpp

```diff
@@ -161,7 +161,8 @@ enum sc_type : int16 {
 	SC_BLIND,
 	SC_BLEEDING,
 	SC_DPOISON, //10
-	SC_COMMON_MAX = 10, // end
+	SC_STONEWAIT,
+	SC_COMMON_MAX = 11, // end
```

Comment:
11 or 12 now?

Reply:
11. It's a trick because SC_STONE and SC_COMMON_MIN are 0. SC_COMMON_MAX is also inclusive.

Reply:
Is it a problem if we make it more obvious by assigning it like SC_COMMON_MAX = SC_STONEWAIT instead then?

Reply:
We can, just need to make sure we update the few cases that use the min and max appropriately!

Reply:
@aleos89 I think you misunderstood Lemongrass3110. He wasn't saying you should change the MIN/MAX values but just change the way they are assigned. So there wouldn't be a need to adjust the checks.

But I'm not sure if that even works in an enum:
```
SC_BLEEDING,
SC_DPOISON, //10
SC_COMMON_MAX = SC_STONEWAIT, //11?
```

Reply:
It works like this:
```
SC_BLEEDING,
SC_DPOISON, //10
SC_STONEWAIT,
SC_COMMON_MAX = SC_STONEWAIT,
```
Then both are 11.